qdb_ts_aggregation_type.java

```diff
@@ -0,0 +1,25 @@
+package net.quasardb.qdb.jni;
+
+public final class qdb_ts_aggregation_type {
```

Comment:
Is there a better way to sync this with the C client API ? I don't like that these magic values are hardcoded in multiple places.